mod_auth/forms.py

```diff
@@ -42,13 +42,13 @@ class LoginForm(Form):
     email = StringField('Email', [DataRequired(
         message='Email is not filled in.')])
     password = PasswordField('Password', [
-        DataRequired(message='Password cannot be empty.')])
+        DataRequired(message='Password cannot be empty.'), Email("entered value is not a valid email address")])
```

Comment:
You probably meant to add it on #L43.
Also, please change it from `entered` to `Entered` while you are at it.